styles.css

```diff
@@ -0,0 +1,31 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap');
+
+body {    
+    background-image: url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    background-color:blueviolet;
+    background-size: 1400px;
+
+    color: white;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    padding: 10px 50px 100px 75px; 
+    
+}
+
+
```

Comment:
I recommend erasing spaces in the blank.

Reply:
Kindly, I suggest you support your comments with why it is good to make them or suggest a solution to the issue.